jvb/src/main/kotlin/org/jitsi/videobridge/cc/BandwidthProbing.kt

```diff
@@ -36,18 +40,28 @@ class BandwidthProbing(
     /**
      * The sequence number to use if probing with the JVB's SSRC.
      */
-    private var seqNum = Random.nextInt(0xFFFF)
+    private val seqNum = Random.nextInt(0xFFFF)
```

Comment:
It looks like this (and ts below) isn't used anywhere other than debug state output, so better to delete it than to make it a `val` (which doesn't make any sense for a sequence number)